common-react/@dbeaver/react-data-grid/src/DataGrid.tsx

```diff
@@ -26,6 +32,7 @@ export interface DataGridProps extends IDataGridCellContext, IDataGridRowContext
   onScroll?: (event: React.UIEvent<HTMLDivElement>) => void;
   onFocus?: (position: ICellPosition) => void;
   onEditorOpen?: (position: ICellPosition) => void;
+  onCellKeyDown?: (args: CellKeyDownArgs<IInnerRow>, event: CellKeyboardEvent) => void;
```

Comment:
Base grid API should not expose the inner implementation (or third-party types and API's) as much as possible. Please change type to something like: `(cell: ICellPosition, event: React.KeyboardEvent<HTMLDivElement>) => void`

Reply:
we need to keep CellKeyboardEvent type to be able to call `event.preventGridDefault();` inside DataGridTable in handler